### ISSUE TITLE
MagicFromCardFilterChoice...is it UI or model

### ISSUE THREAD

**Issue by lodici** [magarena/magarena#1125]
#### Crash report
To help us replicate an issue please include the ``crash.log`` file which is located in the ``/Magarena/logs`` folder.

#### Gameplay report
If raising an issue about strange or unexpected plays during a game, especially decisions made by the AI opponent, please include a ``gameplay-report.zip`` file. You can create a gameplay report by opening the options menu (press Esc during a duel) and selecting the "Gameplay Report" menu item. To attach the file, please drag into the comments box below.

Thanks for your help.

---

**Comment by melvinzhang** [comment]
Agree, passing more "model" level information and leaving it to the UI to decide how best to show it is better than the current low level UI manipulations.
